util/explain_analyze_helper.sql

```diff
@@ -0,0 +1,46 @@
+CREATE OR REPLACE FUNCTION pganalyze.explain_analyze(query text, params text[], param_types text[], analyze_flags text[]) RETURNS text AS $$
+DECLARE
+  prepared_query text;
+  params_str text;
+  param_types_str text;
+  explain_prefix text;
+  explain_flag text;
+  result text;
+BEGIN
+  SET TRANSACTION READ ONLY;
+
+  PERFORM 1 FROM pg_roles WHERE (rolname = current_user AND rolsuper) OR (pg_has_role(oid, 'MEMBER') AND rolname IN ('rds_superuser', 'azure_pg_admin', 'cloudsqlsuperuser'));
+  IF FOUND THEN
+    RAISE EXCEPTION 'cannot run: helper is owned by superuser - recreate function with lesser privileged user';
+  END IF;
```

Comment:
This error occurs in local testing because my Postgres user is a superuser. Maybe the OR should be an AND instead?

For reference, these are the values for the different where clauses:
```sql
select rolsuper, pg_has_role(oid, 'MEMBER'), rolname IN ('rds_superuser', 'azure_pg_admin', 'cloudsqlsuperuser') from pg_roles where rolname = current_user;
 rolsuper | pg_has_role | ?column? 
----------+-------------+----------
 t        | t           | f

Reply:
If you follow the instructions on setting up the separate user ([preview](https://pganalyze-we-query-tuni-nl5vxq.herokuapp.com/docs/query-tuning/collector-workflow#creating-the-pganalyze_explain-role)) it should work as expected, even in a local setup.